blockchain/heightversion.go

```diff
@@ -0,0 +1 @@
+package blockchain
```

Comment:
Empty file.

Reply:
This is based on branch release_v0.3.0, and the issue shouldn't be fixed in this PR.